api/controllers/posts.js

```diff
@@ -22,6 +22,17 @@ const PostsController = {
       res.status(201).json({ message: 'OK', token: token });
     });
   },
+
+  FindById: (req, res) => {
+
+    Post.findOne({_id: req.params.postId}, async (err, post) => {
+      if (err) {
+        throw err;
+      }
+      const token = await TokenGenerator.jsonwebtoken(req.user_id)
```

Comment:
Why do we need to generate another token when we try to fetch a specific post?

Reply:
We followed the same approach the existing functions - create and index - the token is created. It might be interesting to look into it.